fe2-android/app/src/main/java/com/github/dedis/student20_pop/model/network/method/message/data/election/ElectionQuestion.java

```diff
@@ -0,0 +1,114 @@
+package com.github.dedis.student20_pop.model.network.method.message.data.election;
+
+
+import com.github.dedis.student20_pop.model.network.method.message.data.Action;
+import com.github.dedis.student20_pop.model.network.method.message.data.Data;
+import com.github.dedis.student20_pop.model.network.method.message.data.Objects;
+import com.github.dedis.student20_pop.utility.security.Hash;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Collections;
+import java.util.List;
+
+public class ElectionQuestion extends Data {
+
+    private String id;
+    private String question;
+    @SerializedName(value = "voting_method")
```

Comment:
What is the purpose of having a different serialized name ?

Reply:
In the protocol, we use snake_case for naming the variables, as stated in the "Functionality" docs.  Hence, the back-end is expecting a snake_case name for the attributes. On the other hand, the fe2 naming convention is camelCase. To prevent code smells and match the protocol, I have to use the @SerializedName annotation.